code/game/objects/items/ashtray.dm

```diff
@@ -28,7 +28,7 @@
 			var/obj/item/clothing/mask/cigarette/cig = W
 			if(cig.lit == 1)
 				src.visible_message("[user] crushes [cig] in [src], putting it out.")
-				processing_objects.Remove(cig)
+				STOP_PROCESSING(SSobj, cig)
```

Comment:
This probabbbblyyy shouldn't be a thing to be honest. I also believe the cig should stop processing on its own volition in its `Destroy` proc.

Reply:
Lmao yeah wtf out of all things why an actual cigarette